api/envoy/config/metrics/v3/stats.proto

```diff
@@ -359,3 +360,13 @@ message HystrixSink {
   // <https://github.com/Netflix/Hystrix/wiki/Metrics-and-Monitoring#hystrixrollingnumber>`_.
   int64 num_buckets = 1;
 }
+
+// Stats configuration proto schema for built-in *envoy.stat_sinks.circonus* sink.
+// The sink is actually a no-op other than registering a new admin handler
+//
+// Note that only a single sink should be configured.
+// [#extension: envoy.stat_sinks.circonus]
+message CirconusSink {
```

Comment:
Drive by, please put this into an independent proto file. The grab bag we have here is not current best practice.

Reply:
acl